src/translations/en.json

```diff
@@ -323,7 +323,9 @@
         "comfort": "Comfort",
         "home": "Home",
         "sleep": "Sleep",
-        "activity": "Activity"
+        "activity": "Activity",
```

Comment:
This should be on the backend and part of the integration translations. If that's not possible, we should make that support.

Reply:
Thanks for the merge :)
Sadly, there are no translations in https://github.com/home-assistant/home-assistant/tree/dev/homeassistant/components/netatmo that I could update, sorry.